JavaScript Proxy SDK/README.MD

```diff
@@ -0,0 +1,158 @@
+# JavaScript Proxy SDK example
+Create a feature and connect it to client app using the JavaScript Proxy SDK and at the end we play around with some strategies.
+
+## Let's get started
+1. Create a folder then navigate to the folder through your terminal and type:  
+`npm init -y`  
+This should create a file called _package.json_.
+
+2. Create two files:
+
+* index.html
+* app.js
+
+inside _index.html_ paste the code below
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <script src="app.js" type="text/javascript"></script>
+    <title>Unleash Testing</title>
+  </head>
+  <body>
+    <h1>Hello from Unleash Team !</h1>
+  </body>
+</html>
+```
+
+If you open your _index.html_ you should see something like this 👇
+<br/>
+<img src="./static/img/hello-unleash.png" title="index page" />
+
+Now let's install the JavaScript Proxy SDK, run:
+
+```sh
+npm install unleash-proxy-client
+```
+
+Let's import the JavaScript Proxy SDK to our html file:
+
+```html
+<script
+  src="../node_modules/unleash-proxy-client/build/main.min.js"
```

Comment:
I am a bit unsure if we should say something about asset bundling. Wdyt @FredrikOseberg ?

Reply:
Hrm. Yeah ideally we'd show a bundled example with webpack or something similar using the proxy from an external JS file, as most real applications would work. This does get the idea across, but I don't think we should encourage retrieving directly from the node modules folder.

Reply:
Do you think using CDN is a better way to solve it ? 👇
`<script src="https://unpkg.com/unleash-proxy-client@1.5.0/build/main.min.js"/>`
@FredrikOseberg @ivarconr